js/modules/k6/grpc/client_test.go

```diff
@@ -687,7 +693,8 @@ func TestClient(t *testing.T) {
 			val, err := replace(tt.initString.code)
 			assertResponse(t, tt.initString, err, val, ts)
 
-			ctx = lib.WithState(ctx, ts.vuState)
+			mvu.CtxField = lib.WithState(ctx, ts.vuState)
```

Comment:
It should still work without this line, I think :thinking:

Reply:
You're right, done